### PR TITLE
Add python-check-blanket-noqa sample annotations

### PR DESCRIPTION
**Pull request by gliptak**

### CODE REVIEW

File: .pre-commit-hooks.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 -   id: python-check-blanket-noqa
     name: check blanket noqa
-    description: 'Enforce that `noqa` annotations always occur with specific codes'
+    description: 'Enforce that `noqa` annotations always occur with specific codes. Sample annotations: ```# noqa:F401```, ```# NOQA:F401,W203```'
```
[the remark this answers is not in the document]
Comment (gliptak) [reply .pre-commit-hooks.yaml:3]: triple ticks were for formatting in README.md
I thought to highlight that both lower and uppercase are supported
let me know how you would like to see above

[the remark this answers is not in the document]
Comment (asottile) [reply .pre-commit-hooks.yaml:3]: right, the correct way to do inline code is with single ticks, as you can see from earlier in the same line that you edited

upper and lower case are both supported, but the canonical form is lower cased and with a space (trust me, I'm the flake8 maintainer 😉)

[the remark this answers is not in the document]
Comment (gliptak) [reply .pre-commit-hooks.yaml]: thanks for the pointers. updated

File: .pre-commit-hooks.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 -   id: python-check-blanket-noqa
     name: check blanket noqa
-    description: 'Enforce that `noqa` annotations always occur with specific codes'
+    description: 'Enforce that `noqa` annotations always occur with specific codes. Sample annotations: `# noqa: F401`, `# noqa: F401, W203`'
```
Comment (asottile) [inline .pre-commit-hooks.yaml:3]: oops, I only said a space after the colon, not after the period

Comment (gliptak) [reply .pre-commit-hooks.yaml:3]: I'm a little lost here. Do you mean before `Sample`?

Comment (asottile) [reply .pre-commit-hooks.yaml:3]: I told you to change two things, and you changed three things for some reason.  there should not be a space after the comma

**Review by asottile — APPROVED** [review]

**Pull request merged**